splitio/client/client.py

```diff
@@ -429,11 +429,11 @@ def _get_feature_flag_names_by_flag_sets(self, flag_sets):
         :rtype: list
         """
         sanitized_flag_sets = config.sanitize_flag_sets(flag_sets)
-        feature_flags = []
-        [feature_flags.extend(self._split_storage.get_feature_flags_by_set(flag_set)) for flag_set in sanitized_flag_sets]
-        feature_flags_names = []
-        [feature_flags_names.append(feature_flag) for feature_flag in feature_flags]
-        return feature_flags_names
+        feature_flags_by_set = self._split_storage.get_feature_flags_by_sets(sanitized_flag_sets)
+        if feature_flags_by_set is None:
+            _LOGGER.warning("Fetching feature flags for flag set %s encountered an error, skipping this flag set." % (flag_sets))
```

Comment:
there are more than one flag set and you already giving a bunch of warning on the sanitized. still the log in unaccurate it should say "method_called (which can be get_treatment_by_flag_set or get_treatments_by_flag_sets:, etc"